### PR TITLE
Added chart repo

### PR DESCRIPTION
**Pull request by goruha**
## What
* Umbrella chart for chartmuseum.
* Use one chartmuseum subchart to serve
* Use second chartmuseum subchart as push gateway

## Why
* Allow to make private repo with different access to pull and to push
* Based on official chartmuseum

### CODE REVIEW

File: incubator/chart-repo/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (osterman) [inline incubator/chart-repo/LICENSE]: Fix

File: incubator/chart-repo/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```
Comment (osterman) [inline incubator/chart-repo/Chart.yaml]: Update description

File: incubator/chart-repo/LICENSE (hunk 1 above)
Comment (goruha) [reply incubator/chart-repo/LICENSE]: Done

File: incubator/chart-repo/Chart.yaml (hunk 2 above)
Comment (goruha) [reply incubator/chart-repo/Chart.yaml]: Done

**Pull request merged**